lib/openssl.cmake

```diff
@@ -23,7 +25,7 @@ if (USE_CRYPTO_SERVICE)
         message(FATAL_ERROR
           "Could not figure out config for cross compiling openssl. "
           "Please set openssl_config manually, by following the instructions in: "
-          "https://github.com/openssl/openssl/blob/openssl-3.0.0/INSTALL.md#manual-configuration"
+          "https://github.com/openssl/openssl/blob/master/INSTALL.md#manual-configuration"
```

Comment:
Maybe use https://github.com/openssl/openssl/blob/OpenSSL_1_1_1s/INSTALL instead? Not sure which part of that file you had to read to get OpenSSL 1.1.1s compiling though!

Reply:
Good point.